coxeter/shapes/polyhedron.py

```diff
@@ -356,9 +356,33 @@ def vertices(self):
 
     @property
     def faces(self):
-        """list(:class:`numpy.ndarray`): Get the polyhedron's faces."""
+        """list(:class:`numpy.ndarray`): Get the polyhedron's faces.
+
+        Results returned as vertex index lists.
+        """
         return self._faces
 
+    @property
+    def edges(self):
+        """list(:class:`numpy.ndarray`): Get the polyhedron's edges.
+
+        Results returned as vertex index pairs.
+        """
+        edges = []
+        for face in self.faces:
+            [
+                edges.append((i, j))
+                for i, j in zip(np.append(face, face[0]), np.append(face[1:], face[0]))
+                if (j, i) not in edges
+            ]
+        return edges
+
+    @property
+    def edge_vectors(self):
+        """list(:class:`numpy.ndarray`): Get the polyhedron's edges as vectors."""
+        edges = np.asarray(self.edges)
+        return list(self.vertices[edges[:, 1]] - self.vertices[edges[:, 0]])
```

Comment:
Why return this as a `list`? Ideally we can keep everything as NumPy arrays unless it's a ragged array (e.g. faces may have different numbers of vertices in them).

Also, because `self.vertices` is a NumPy array, you don't have to convert `edges`. You can index it with a list and it'll do fancy NumPy indexing.

```suggestion
        edges = self.edges
        return self.vertices[edges[:, 1]] - self.vertices[edges[:, 0]]
```